postman/p4pa-auth-E2E.postman_collection.json

```diff
@@ -397,7 +398,113 @@
 									"});"
 								],
 								"type": "text/javascript",
-								"packages": {}
+								"packages": {},
+								"requests": {}
+							}
+						}
+					],
+					"request": {
+						"auth": {
+							"type": "bearer",
+							"bearer": [
+								{
+									"key": "token",
+									"value": "{{accessToken}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{p4paAuthBaseUrl}}/oauth/userinfo",
+							"host": [
+								"{{p4paAuthBaseUrl}}"
+							],
+							"path": [
+								"oauth",
+								"userinfo"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "01.postTokenLimited",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Auth - 01.postTokenLimited - Responses with 200\", function () {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"pm.test(\"Auth - 01.postTokenLimited - Verify response body\", function () {",
+									"    let jsonResponse = pm.response.json();",
+									"    pm.expect(jsonResponse).have.property(\"token_type\").to.eq(\"bearer\");",
+									"    pm.expect(jsonResponse).have.property(\"expires_in\").to.eq(14400);",
+									"    pm.expect(jsonResponse).have.property(\"access_token\");",
+									"",
+									"    pm.collectionVariables.set(\"accessTokenLimited\", jsonResponse.access_token)",
+									"});"
+								],
+								"type": "text/javascript",
+								"packages": {},
+								"requests": {}
+							}
+						}
+					],
+					"request": {
+						"auth": {
+							"type": "bearer",
+							"bearer": [
+								{
+									"key": "token",
+									"value": "{{accessToken}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n  \"organizationId\": 2,\n  \"app\": \"test\",\n  \"resource\": \"resource\",\n  \"resourceId\": \"resourceId\",\n  \"expireInSeconds\": 14400,\n  \"singleUsage\": false\n}",
```

Comment:
this collection would be run also on UAT... does this organization exists there? otherwise, you could modify the previous call to getUserInfo to store one of the organization id